drizzlepac/haputils/poller_utils.py

```diff
@@ -127,7 +127,6 @@ def interpret_obset_input(results, log_level):
     """
```

Comment:
Please add documentation regarding the input change.  I realize this section does not discuss the single column input, and therefore, also does not discuss the two column input.  You do not need to be elaborate, but please add a bit of documentation.  For the one column input you can just note how information is extracted from the headers.  For the two column input please show how the obs_info_dict changed, particularly as the change is in the middle of the "info" string, so the existing examples will have to be updated.